storey/drivers.py

```diff
@@ -223,7 +224,11 @@ def _build_feature_store_update_expression(self, aggregation_element, additional
             for name, value in additional_data.items():
                 if name.casefold() in self.saved_engine_words.keys():
                     name = f'`{name}`'
-                expressions.append(f'{name}={self._convert_python_obj_to_expression_value(value)}')
+                expression_value = self._convert_python_obj_to_expression_value(value)
+                if expression_value:
+                    expressions.append(f'{name}={self._convert_python_obj_to_expression_value(value)}')
+                else:
+                    expressions.append(f'REMOVE {name}')
```

Comment:
Maybe add a test for this case?

Reply:
this is "moshe". it's time is being deleted in the test